src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -22,9 +22,11 @@ dotnet add package OpenTelemetry.Instrumentation.Runtime
 
 ### Step 2: Enable runtime instrumentation
 
-Runtime instrumentation should be enabled at application startup. This is
-typically done in the `ConfigureServices` of your `Startup` class. The example
-below enables this instrumentation by using an extension method on
+Runtime instrumentation should be enabled at application startup.
```

Comment:
you can keep this doc very simple by explaining the extension method and linking to the examples. Does not have to deal with asp.net core vs others here.

Reply:
I realized that the doc for [OpenTelemetry.Extensions.Hosting](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Extensions.Hosting) doesn't have any info on the `AddOpenTelemetryMetrics` extension, or how to configure metrics on ASP.NET Core applications.

Instead, I can only find it in the README of [Prometheus Exporter AspNetCore](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Prometheus.AspNetCore/README.md#step-2-configure-opentelemetry-meterprovider).

Reply:
Perhaps it's better to deal with the asp.net core vs others in the read me  of OpenTelemetry.Extensions.Hosting.

Reply:
yes. Its not the concern of this runtime instrumentation that other components are not well documented.